tests/5.0/parallel_master_taskloop/test_parallel_master_taskloop.F90

```diff
@@ -0,0 +1,58 @@
+!/===--- test_parallel_master_taskloop.F90 -------------------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! This test checks the parallel master taskloop directive. The test performs
+! simple operations on an int array which are then checked for correctness.
+!
+!/===----------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_parallel_master_taskloop
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+
```

Comment:
I think you want to add:
```suggestion

    OMPVV_TEST_OFFLOADING
```

Reply:
Added OMPVV_TEST_OFFLOADING to both C and Fortran versions.